src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -366,13 +263,17 @@ public String getInvalidityInfoForEmail(String email) {
         String sanitizedValue = Sanitizer.sanitizeForHtml(email);
 
         if (email.isEmpty()) {
-            return String.format(EMAIL_ERROR_MESSAGE, email, REASON_EMPTY);
+            return getInterpolatedErrorMessage(
+                    EMAIL_ERROR_MESSAGE, email, EMAIL_FIELD_NAME, REASON_EMPTY, EMAIL_MAX_LENGTH);
```

Comment:
Here (and in many parts after here) it should be fine to include the parameters in the same line since it's not squished at all:
```java
return getInterpolatedErrorMessage(EMAIL_ERROR_MESSAGE, email, ...,
                                   ...);
```